ipinfo_db/client.py

```diff
@@ -10,6 +10,13 @@
 class Client:
 
     def __init__(self, access_token=None, path=None, replace=False):
+        f'''ipinfo_db handler method.
+
+        :param access_token: Required. Type: str. IPinfo access token to download the IP to Country ASN database.
```

Comment:
`access_token` can be optional in case `path` is provided.

Reply:
@rm-Umar Got it. Updated docstring [here](https://github.com/ipinfo/python-db/pull/2/commits/4066c85c05e20265932005eafe8f0d2f168f21f9#diff-8a2406f588ed315828009b48b086fb6d3e707fc50b1c3a2af3959bb8fc9ed414R15). Please review.

Reply:
Still doesn't feel right. You know better about the docs but what about if we just make it 
```
:param access_token: Optional. Type: str. IPinfo access token to download the IP to Country ASN database.
                                     Required in case of data download.
```
or do we need to document the whole thing? explaining other details too?

Reply:
@rm-Umar Thank you for the suggestion. I like it. I have made the [change](https://github.com/ipinfo/python-db/pull/2/commits/aa65984bbfd80e0261c7eb51d123beb8fef3bc59#diff-8a2406f588ed315828009b48b086fb6d3e707fc50b1c3a2af3959bb8fc9ed414R15).

I can provide the other details on the community docs. Thanks!